src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -46,7 +46,7 @@ limitations under the License.
     <field name="CoolSetpointPresent" mask="0x02"/>
   </bitmap>
 
-  <enum name="ThermostatSystemMode" type="ENUM8">
+  <enum name="ThermostatSystemModeEnum" type="ENUM8">
```

Comment:
This ended up being called SystemModeEnum in the spec, and the XML has been updated to that.